.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,40 @@
+name: Dependabot update node-modules
+on: pull_request
```

Comment:
How about `branches: '**dependabot**'` ? That avoids unnecessary runs for all other PRs, and should replace the `if actor` below.

Reply:
That could work, and allows us to keep amending and maybe getting rebasing working.